refinery/tool_manager/migrations/0014_auto_20170419_1509.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tool_manager', '0013_auto_20170419_1500'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='tool',
+            name='container_name',
+            field=models.CharField(default='', unique=True, max_length=250, blank=True),
+            preserve_default=False,
+        ),
+        migrations.AlterField(
+            model_name='tooldefinition',
+            name='container_input_path',
+            field=models.CharField(default='', max_length=500, blank=True),
+            preserve_default=False,
+        ),
+        migrations.AlterField(
+            model_name='tooldefinition',
+            name='galaxy_workflow_id',
+            field=models.CharField(default='', max_length=250, blank=True),
+            preserve_default=False,
+        ),
+    ]
```

Comment:
I could, but this requires making sure that everyone has applied the prior migrations properly before I'm able to remove them. 

Django docs say:
```
You are encouraged to make migrations freely and not worry about how many you have; the migration code is optimized to deal with hundreds at a time without much slowdown. However, eventually you will want to move back from having several hundred migrations to just a few, and that’s where squashing comes in.
```